README.md

```diff
@@ -1,18 +1,18 @@
 # XInTheLoop Library
 
-XInTheLoop is a Modelica library for interfacing a model with an external system for continous interaction during simulation of the model. The external system in this case is a software tightly coupled with e.g. a physical process hardware, some user interface, another model simulation, a computational service, or some other software service.
+XInTheLoop is a Modelica library for interfacing a model with an external system for continuous interaction during simulation of the model. The external system in this case is a software tightly coupled with e.g. a physical process hardware, some user interface, another model simulation, a computational service, or some other software service.
 
 A generic UDP based protocol is developed to exchange values between the Modelica model and the external system in both directions at regular intervals.
 
 Together with a set of generic support blocks, it should be easy to implement your own blocks for interfacing one particular external system.
 
-The external system can be developed in any language/framework that support UDP communication. An included Python script for debugging the UDP communication can be used as inspiration.
+The external system can be developed in any language/framework that support UDP communication. An [included Python script](XInTheLoop/Resources/tools/site1-protocol.py) for debugging the UDP communication can be used as inspiration.
 
 ## Site1 Example
 
 The Site1 example is a test model and interfacing blocks for one particular external FCS system containing a fuel cell, a DC-to-DC-converter, a battery, and a variable load.
 
 ## Dependencies
 
-- Modelica v3.2.3
-- Modelica_DeviceDrivers v1.7.1
+- Modelica v4.0.0
+- Modelica_DeviceDrivers v2.1.0
```

Comment:
> I suggest changing this to Modelica_DeviceDrivers v2.1.1, since OM loads this version anyway.

I don't think it's correct that OM loads Modelica_DeviceDrivers v2.1.1 anyway. If Modelica_DeviceDrivers v2.1.0 is installed by the OM packet manager at the time when loading XInTheLoop, then I assume OM will use v2.1.0. See also my other comment about this version dependency. XInTheLoop doesn't really require Modelica_DeviceDrivers v2.1.1.